service/src/main/java/uk/gov/hmcts/reform/fpl/model/AdditionalChild.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.UUID;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AdditionalChild {
```

Comment:
Please make it generic. That object is in fact CCD wrapper for lists. One generic instance should have been used for all lists, here for both lists of children and list of respondents.